plugin/storage/badger/spanstore/read_write_test.go

```diff
@@ -607,14 +607,14 @@ func BenchmarkServiceIndexLimitFetch(b *testing.B) {
 
 // Opens a badger db and runs a test on it.
 func runLargeFactoryTest(tb testing.TB, test func(tb testing.TB, sw spanstore.Writer, sr spanstore.Reader)) {
-	assert := assert.New(tb)
+	assertion := require.New(tb)
```

Comment:
This is the last other different change . Create require assertions instead of assert